drivers/starburst/src/metabase/driver/implementation/execute.clj

```diff
@@ -187,16 +191,29 @@
         (.setFetchDirection stmt ResultSet/FETCH_FORWARD)
         (catch Throwable e
           (log/debug e (trs "Error setting prepared statement fetch direction to FETCH_FORWARD"))))
-      (sql-jdbc.execute/set-parameters! driver stmt params)
-      (proxy [java.sql.PreparedStatement] []
-        (executeQuery []
-          (try
-            (let [rs (.executeQuery stmt)]
-              (remove-impersonation conn)
-              rs)
-              (catch Throwable e (handle-execution-error e))))
-        (setMaxRows [nb] (.setMaxRows stmt nb))
-        (close [] (.close stmt)))
+      (let [ps (proxy [java.sql.PreparedStatement] []
+          (executeQuery []
+            (try
+              (let [rs (.executeQuery stmt)]
+                (remove-impersonation conn)
+                rs)
+                (catch Throwable e (handle-execution-error e))))
+          (setMaxRows [nb] (.setMaxRows stmt nb))
+          (setObject
+            ([index obj] (.setObject stmt index obj))
+            ([index obj type] (.setObject stmt index obj type)))
+          (setTime
+            ([index val] (.setTime stmt index val))
+            ([index val cal] (.setTime stmt index val cal)))
+          (setTimestamp
+            ([index val] (.setTimestamp stmt index val))
+            ([index val cal] (.setTimestamp stmt index val cal)))
+          (setDate
+            ([index val] (.setDate stmt index val))
+            ([index val cal] (.setDate stmt index val cal)))
+          (close [] (.close stmt)))]
```

Comment:
Are these the only methods invoked by Metabase?

Reply:
That's the only ones I have found (save `getParameterMetaData`), both by running all the tests and looking at the Metabase source code.

Reply:
I added several other methods. Not sure if you want more.